### ISSUE TITLE
Privacy Guidline

### ISSUE THREAD

**Issue by arist0v** [createcandle/candleappstore#2]
I would enjoy if you could provide a privacy guidline to help provide and class privacy in your store.

there is 1 to 5 closed eye, but no information on the critera used to define each level of privacy

**Comment by createcandle** [comment]
Yes, this is a good point. Where to place this though.. in the Candle app store, or on the website? I'm edging to in the app store.

**Comment by arist0v** [comment]
this should be on the website, maybe create a developper section where we could put all information about that subject

**Comment by arist0v** [comment]
https://www.candlesmarthome.com/get-involved 

this page look like the good place to add a contribute section wich link to multiple articles(how to guidelines and stuff)
